scripts/core/exp-command-buffer.yml

```diff
@@ -0,0 +1,307 @@
+#
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
```

Comment:
We have updated the project license to `Apache-2.0 WITH LLVM-exception`,
```suggestion
#
# Copyright (C) 2022 Intel Corporation
#
# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
# See LICENSE.TXT
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Reply:
Apologies this should be fixed now!